community/sigla/about.en.md

```diff
@@ -10,14 +10,18 @@ old_url: http://www.rism.info/en/sigla/about-sigla.html
+RISM sigla are composed of a country abbreviation, a city abbreviation, and (usually) an institution abbreviation. The country abbreviations are taken from the [Vehicle Registration Identification code](https://en.wikipedia.org/wiki/International_vehicle_registration_code). The city identifier is assigned by RISM and is unique to a country. It may be helpful to look at a few examples.
 
-For example:\
-F-Pn = **F**rance, **P**aris, Bibliothèque **n**ationale de France, Département de la Musique.
+In **I-Rc**, **I** stands for Italy, **R** is for Rome, and **c** is for the [Biblioteca Casanatense](https://rism.online/institutions/30000230).
 
-In the case of private collections, the entire last name of the owner is added to the city code:  
-I-PEbattisti = **I**taly, **Pe**rugia, Biblioteca privata Renzo **Battisti**.
+In **US-PHf**, **US** stands for the United States, **PH** is for Philadelphia, and **f** is for the [Parkway Central Library, Music Department](https://rism.online/institutions/30002414).
 
-The library sigla are administered in a database maintained by the RISM Editorial Center in Frankfurt am Main. Each year, the RISM office receives a couple hundred inquiries on average for institutions that do not yet have a siglum. In such cases, RISM consults with the national group in question and assigns a new siglum.
+Some institutions that are prominent in the city omit the institution abbreviation. In **D-B**, the **D** stands for Germany, and the **B** for Berlin. The institution is implied as the [**Staatsbibliothek zu Berlin**](https://rism.online/institutions/30000655) (The Berlin State Library). All other institutions in Berlin will have a institution abbreviation.
 
-Contact us to request a new siglum or to inform us of any incorrect information: [contact@rism.info](mailto:contact@rism.info)
+Some large institutions may have more than one siglum assigned. **F-Pn** refers specifically to the [Music Department at the Bibliothèque nationale de France](https://rism.online/institutions/30001488), while **F-Pnm** refers to the [Manuscripts department](https://rism.online/institutions/30078937) at the same institution. We cannot always track where documents are actually held in an institution, so it is best to confirm with the institution directly.
+
+Private collections may also have a siglum assigned. The last name of the owner or collector is generally used as the institution component in this case. In **CDN-Mfisher**, the **CDN** stands for Canada, the **M** for Montreal, and **fisher** for the [private collection of Sidney T. Fisher](https://rism.online/institutions/30000470). Since individuals and collections can move, the city name does not always correspond to the location of the collection. In this case, the collection is now held in Toronto at the [University of Toronto](https://rism.online/institutions/30000488) (**CDN-Tu**).
```

Comment:
```suggestion
Private collections may also have a siglum assigned. The last name of the owner or collector is generally used as the institution component in this case. In **CDN-Mfisher**, the **CDN** stands for Canada, the **M** for Montreal, and **fisher** for the [private collection of Sidney T. Fisher](https://rism.online/institutions/30000470). Since individuals and collections can move, the city name does not always correspond to the current location of the collection. In this case, the collection is now held in Toronto at the [University of Toronto](https://rism.online/institutions/30000488) (**CDN-Tu**).
```